connectors/protocol/connectors.py

```diff
@@ -638,13 +638,24 @@ async def update_last_sync_scheduled_at_by_job_type(self, job_type, new_ts):
             case _:
                 raise ValueError(f"Unknown job type: {job_type}")
 
-    async def sync_starts(self):
+    async def sync_starts(self, job_type=JobType.FULL):
+        last_sync_information = (
+            {
+                "last_access_control_sync_status": JobStatus.IN_PROGRESS.value,
+                "last_access_control_sync_error": None,
+            }
+            if job_type == JobType.ACCESS_CONTROL
```

Comment:
Nit: This type of structure ("if condition, else default") is going to be more bug prone than one that explicitly disallows unknown states ("if condition_a, else if condition_b, else error"). If we add a 3rd sync type, and someone doesn't notice this code, things will keep working, but with unexpected behavior. I prefer to ensure that we fail-fast if our assumptions ever stop holding.

Reply:
Makes sense, adjusted it 👍